### DISCUSSION TITLE
AUDIO THROUGH CV/GATE OUTS?

### DISCUSSION THREAD

**Discussion by worker-gray** [SynthstromAudible/DelugeFirmware#1641] [Ideas]
One of the major choke-points of multitracking the deluge is the stereo pair output. I don't know if it's been suggested or if it's even possible, but it would be very helpful to the 'non-modular' crowd to be able to route tracks through the CV and GATE outputs for individual recording and/or effects-- mono or stereo split.
I would suggest extra routing options be accessed via a submenu under panning.

**Comment by sapphire-arches** [comment]
Unfortunately the CV DAC is too slow for audio and the gates outs are digital only. The gate outs are actually using pins that could support sending digital audio data (SSI clock, transmit, and word select for SSI unit 3) but external hardware would have to be built to turn that in to analog audio.

**Reply by worker-gray** [reply]
thanks for the info!

**Comment by mcpepe** [comment]
Maybe these CV outs (or trigs out) could be used to output a simple metronome sound for a click track? This metronome would not need audio quality
